google-cloud-edge_container-v1/.owlbot.rb

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# OwlBot script for google-cloud-edge_container-v1
+OwlBot.modifier path: "proto_docs/google/cloud/edgecontainer/v1/resources.rb" do |content|
```

Comment:
Can you notate in a comment that this is due to `https://github.com/googleapis/gapic-generator-ruby/issues/824`? That'll help us identify these so we can remove them once we actually fix that issue.